store/store.go

```diff
@@ -71,15 +71,14 @@ func (a *Alerts) Run(ctx context.Context, interval time.Duration) {
 
 func (a *Alerts) gc() {
 	a.Lock()
-	defer a.Unlock()
-
 	var resolved []*types.Alert
```

Comment:
We need to stop passing `[]*types.Alert` and instead now send a slice of values to the callback.

```suggestion
	var resolved []types.Alert
```

Reply:
I can submit a separate PR for this